playbook/package.json

```diff
@@ -64,6 +64,8 @@
     "moment": "^2.29.1",
     "moment-strftime": "^0.5.0",
     "moment-timezone": "^0.5.26",
+    "node-sass": "^4.12.0",
+    "prop-types": "^15.6.2",
```

Comment:
```suggestion
```
If that's right and these should just be dev dependencies, I'll just delete the two lines